docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -0,0 +1,105 @@
+# Performing Device Firmware Upgrade in nRF Connect SDK examples
+
+The following examples for the development kits from Nordic Semiconductor
+support over-the-air Device Firmware Upgrade:
+
+-   [CHIP nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md)
+-   [CHIP nRF Connect Lighting Example Application](../../examples/lighting-app/nrfconnect/README.md)
+
+The
+[CHIP nRF Connect Pigweed Example Application](../../examples/pigweed-app/nrfconnect/README.md)
+does not support DFU.
+
+Currently the Bluetooth LE is the only available transport for performing Device Firmware Upgrade operation and for that purpose it uses [Simple Management Protocol](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#device-mgmt). Upgrade can be done either using smartphone application or PC command line tool.
+
+## Device Firmware Upgrade using smartphone
+
+To upgrade your device firmware over Bluetooth LE using smartphone, complete the following tasks:
+
+1. Install on your smartphone [nRF Connect for Mobile](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-mobile) or [nRF Toolbox](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Toolbox) application.
+2. Push **Button 1** on the device to enable software update functionality.
+3. Push **Button 4** on the device to start Bluetooth LE advertising.
+4. Follow the instructions in the section about downloading the new image to a device on the
+[FOTA upgrades](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf52.html#fota-upgrades)
+page in the nRF Connect documentation.
+
+## Device Firmware Upgrade using PC command line tool
+
+To upgrade your device firmware over Bluetooth LE, you can use PC command line tool provided by the [mcumgr](https://github.com/zephyrproject-rtos/mcumgr) project.
+
+> **_WARNING:_** mcumgr tool using Bluetooth LE is available only for Linux and macOS systems.
+> On Windows there is no support for Device Firmware Upgrade over BLE yet.
+Complete the following steps to perform DFU using mcumgr:
+
+1. Follow the [mcumgr command line tool installation](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#command-line-tool) instruction to install the tool.
+2. Push **Button 1** on the device to enable software update functionality.
+3. Push **Button 4** on the device to start Bluetooth LE advertising.
+4. Navigate to your example directory and call the following command:
```

Comment:
What about remove this section and provide a path to `build/zephyr/app_update.bin` in line 5?

One for simplicity, but also in case someone would like to recompile the application, the `cd ../../` command would be needed.